tests/lib/rules/arrow-body-style.js

```diff
@@ -72,6 +84,7 @@ ruleTester.run("arrow-body-style", rule, {
         },
         {
             code: "var foo = () => { return bar(); };",
+            output: "var foo = () =>   bar() ;",
```

Comment:
The two extra spaces are from after the arrow, and after the opening curly. At the moment, the fixer keeps all text between the tokens in case it contains comments, newlines, etc. I think people can use `no-multi-spaces` and `semi-spacing` if they don't want the extra spaces.

Would you recommend that we special-case the fixes where there are only space characters  between these tokens?